molgenis-dataexplorer/src/main/java/org/molgenis/dataexplorer/controller/DataExplorerController.java

```diff
@@ -182,9 +183,20 @@ else if (moduleId.equals("annotators"))
 			model.addAttribute("annotationRun", annotationRun);
 			model.addAttribute("entityName", entityName);
 		}
+
 		return "view-dataexplorer-mod-" + moduleId; // TODO bad request in case of invalid module id
 	}
 
+	@RequestMapping(value = "/copy", method = GET)
+	@ResponseBody
+	public boolean showCopy(@RequestParam("entity") String entityName)
```

Comment:
This exact information is already available in the REST api under /api/v2/entityName. If you have no right to read items, you get a not-authorized status code and the response shows in meta.writable if the repository is writable.